README.md

```diff
@@ -23,35 +28,36 @@
 
 1. **Create an environment File**
 
-   Create a `.env.development` and `.env.production` files and define environment variables as below. Refer: [https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
+   Create a `.env.development` (local) or `.env.production` (prod) file and define environment variables as below. 
 
    For local:
 
    ```
    GATSBY_SITE_PATH_PREFIX = Demo
    
-   # Personal access token with read access created from github as per instructions [here](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line)
+   # Personal access token with read access created from Github Enterprise
```

Comment:
Maybe we can link the `Github enterprise` part to https://git.corp.adobe.com?

Reply:
Done!